### ISSUE TITLE
Bug appears when using 3.1.1. 

### ISSUE THREAD

**Issue by petrbrzek** [wwayne/react-tooltip#158]
This didn't happen with the previous version.

```
TypeError: Cannot read property 'l' of undefined
  at getTipOffsetLeft (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/avocode-ui/node_modules/react-tooltip/dist/utils/getPosition.js:35:40)
  at exports.default (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/avocode-ui/node_modules/react-tooltip/dist/utils/getPosition.js:170:22)
  at ReactTooltip.updatePosition (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/avocode-ui/node_modules/react-tooltip/dist/index.js:398:46)
  at ReactTooltip.<anonymous> (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/avocode-ui/node_modules/react-tooltip/dist/index.js:320:20)
  at CallbackQueue.notifyAll (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/react/lib/CallbackQueue.js:65:22)
  at ReactUpdatesFlushTransaction.close (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/react/lib/ReactUpdates.js:57:24)
  at ReactUpdatesFlushTransaction.closeAll (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/react/lib/Transaction.js:202:25)
  at ReactUpdatesFlushTransaction.perform (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/react/lib/Transaction.js:149:16)
  at ReactUpdatesFlushTransaction.perform (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/react/lib/ReactUpdates.js:86:38)
  at Object.flushBatchedUpdates (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/react/lib/ReactUpdates.js:147:19)
  at Object.wrapper [as flushBatchedUpdates] (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/react/lib/ReactPerf.js:66:21)
  at ReactDefaultBatchingStrategyTransaction.closeAll (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/react/lib/Transaction.js:202:25)
  at ReactDefaultBatchingStrategyTransaction.perform (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/react/lib/Transaction.js:149:16)
  at Object.batchedUpdates (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/react/lib/ReactDefaultBatchingStrategy.js:62:19)
  at Object.enqueueUpdate (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/react/lib/ReactUpdates.js:176:22)
  at enqueueUpdate (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/react/lib/ReactUpdateQueue.js:24:16)
  at Object.enqueueCallback (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/react/lib/ReactUpdateQueue.js:108:5)
  at ReactTooltip.ReactComponent.setState (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/react/lib/ReactComponent.js:67:18)
  at updateState (/Users/petrbrzek/Desktop/dev/forked-repos/avocode/node_modules/avocode-ui/node_modules/react-tooltip/dist/index.js:315:18)
```

**Comment by wwayne** [comment]
Weird, that should be fixed in 3.1.1, can you show me more about your code?

**Comment by cesarwbr** [comment]
You can see this problem in sample page: http://wwayne.com/react-tooltip/
I think the error happens when you scroll during the tooltip

**Comment by wwayne** [comment]
Hm, I see it, thanks, will have a deep investigation

**Comment by maullerz** [comment]
Have this error also,
in #150 when i use `delayShow` and remove cursor from element while tooltip didnt appear yet.

**Comment by caesarsol** [comment]
I'm having the same problem here, exactly described by @maullerz.
I've found that `state.place === ""`, that causes the error in `getTipOffsetLeft()` when called with empty string.

**Comment by wwayne** [comment]
I picked a wrong way to fix https://github.com/wwayne/react-tooltip/issues/143, this should be fixed in 3.1.2
